### PR TITLE
adding guide for running custom qt5webapp using udocker

### PR DESCRIPTION
**Pull request by katakombi**
Hello everybody,

 I've created a guide how to set up a custom Qt5-based web application using udocker and user space containers. I'd love to get the guide into here so tell me what you still need to make it comply.

cheers Stefan

### CODE REVIEW

**Comment by SalocinHB** [comment]
Looks promising. Please have a look at our [style guide](https://github.com/Uberspace/lab/blob/master/STYLE.md).

**Comment by jens1o** [comment]
any update?

File: source/guide_udocker.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+::
+
+ [isabell@stardust ~] WWW_PORT=$(( $RANDOM % 4535 + 61000)); netstat -tulpen | grep $WWW_PORT && echo "try again"
+
+Usually this gives you a free port on first try. If you see `try again` repeat the previous command.
+Now we adapt the settings so that the external port 443 is being mapped to the local port.
+
+::
+
+ [isabell@stardust ~] cat << EOF > ~/html/.htaccess
```
Comment (SalocinHB) [inline source/guide_udocker.rst]: please avoid `cat << EOF >` and similar stuff, just ask the user to edit the file with an editor of their choice.

**Comment by katakombi** [comment]
I've erased my local build env and I've let expire my associated uberspace. Thus can't preview or test quick changes right now. I'm going to apply the changes as soon as I've got some time. Turns out to be more difficult than I thought to stick to your syle guide since I've got my own way to write copy-n-pastable HOWTOs, and to make it compliant afterwards seems to be more difficult than redoing it from scratch :)

**Comment by nichtmax** [comment]
Feel free to open a new MR when you're ready!

**Comment by jorsn** [comment]
If you reopen this, I think you probably also need to set `PROOT_NO_SECCOMP=1`, because
udocker ignores P2 exec mode (see https://github.com/indigo-dc/udocker/issues/274). I needed it in #656 and #160.

**Pull request closed without merging**